options/auto_retry_options.go

```diff
@@ -0,0 +1,18 @@
+package options
+
+const DEFAULT_MAX_RETRY_ATTEMPTS = 3
+const DEFAULT_SLEEP = 5
+
+// List of recurring transient errors encountered when calling terraform
+// If any of these match, we'll retry the command
+var RETRYABLE_ERRORS = []string{
```

Comment:
Does the list of expressions look reasonable? I think I covered all the items in https://github.com/gruntwork-io/terragrunt/issues/388

Reply:
Yup!